parachains/runtimes/collectives/collectives-polkadot/src/constants.rs

```diff
@@ -26,6 +26,9 @@ pub mod account {
 	/// It is used as a temporarily place to deposit a slashed imbalance
 	/// before the teleport to the Treasury.
 	pub const REFERENDA_PALLET_ID: PalletId = PalletId(*b"py/refer");
+	/// The Polkadot Fellowship pallet ID
+	/// The identifier determines an account address of the collective.
+	pub const FELLOWSHIP_PALLET_ID: PalletId = PalletId(*b"py/fella");
```

Comment:
not really a `pallet id`, but `pot id`.
probably worth to introduce a `PotId` type.

Reply:
I think the impl of `AccountIdConversion` is requiring `Decode` unnecessarily in some cases.  
It should be possible to just use `&'static str` or so, but is not because of that.